Dockerfile

```diff
@@ -35,5 +50,13 @@ RUN chmod +x /PrairieLearn/docker/init.sh \
     && mv /PrairieLearn/docker/config.json /PrairieLearn \
     && mkdir /course
 
+# r-requirements.R is copied in the above setup.
```

Comment:
Can you shift this up and combine with the earlier block that does `# Install Python/NodeJS dependencies`? Docker produces better images if we have fewer RUN commands and if they are ordered so that ones that change more frequently come later (so PL core code should be later than dependencies).

Reply:
Done. This is now here: 
https://github.com/PrairieLearn/PrairieLearn/pull/1023/commits/12798b271b819f07809cc2b0c249b2d2d5a9caae#diff-03f6ff77580eb0ea364c4f941578c252